stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channels/info/SelectedChannelMenu.kt

```diff
@@ -138,12 +138,20 @@ internal fun DefaultSelectedChannelMenuHeaderContent(
     Text(
         modifier = Modifier.fillMaxWidth(),
         textAlign = TextAlign.Center,
-        text = selectedChannel.getMembersStatusText(LocalContext.current, currentUser),
+        text = selectedChannel.getMembersStatusText(
+            context = LocalContext.current,
+            currentUser = currentUser,
+            countCurrentUserAsOnlineMember = ChatTheme.userPresence.currentUser.countAsOnlineMember,
+            countOtherUsersAsOnlineMembers = ChatTheme.userPresence.otherUsers.countAsOnlineMember,
```

Comment:
Would be good to receive as parameter the `UserPresence` object, instead of multiple flags